examples/benchmarks/tablebench/src/azure.ts

```diff
@@ -36,6 +36,7 @@ export async function initFluid() {
 		({ container } = await client.createContainer(containerSchema));
 
 		// TODO: Waiting for 'attach()' is a work around for https://dev.azure.com/fluidframework/internal/_workitems/edit/6805
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-return
 		await container.attach().then((containerId) => (location.hash = containerId));
```

Comment:
Why disable and not fix? Can we add a comment?

Reply:
Fixing this required adding a dependency on fluid-static, which seems wrong but nothing complained about it, so I guess it's fine.